packages/ui-causal-graph-editor/src/shared/graph-layout/planar-layout.tsx

```diff
@@ -47,7 +56,7 @@ class PlanarLayoutBuilder extends GraphLayoutBuilder<PlanarLayout> {
  *
  * @param data the data of a pure node or link
  */
-function getNodeValue(data: any): number {
+function getNodeValue(data: SugiNodeDatum<{ ord?: number }> | SugiLinkDatum<{ ord?: number }>): number {
```

Comment:
Does the `getNodeValue` name make sense if the argument can be either a node or a link?

Reply:
Agree not a great name, have updated to be getOrdValue. At this stage from what I understand link is essentially a fake node which is used to help draw the curved edges :)